dbms/src/Storages/DeltaMerge/Proto/stable.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package DB.DM.PB;
+
+message StableMeta {
+    uint64 valid_rows = 1;
+    uint64 valid_bytes = 2;
+    uint64 num_files = 3;
+    repeated uint64 file_ids = 4;
```

Comment:
In order to support lightweight ingesting optional index, we should make this field to be something like

`repeat struct {uint64 file_id, uint64 file_epoch}`

Reply:
Change to:

```proto
message FileIdentifier {
    uint64 file_id = 1;
}

message StableLayerMeta {
    uint64 valid_rows = 1;
    uint64 valid_bytes = 2;
    uint64 num_files = 3;
    repeated FileIdentifier file_ids = 4;
}
```

And do not add `file_epoch` in this PR.